src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -220,11 +220,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		// 'applicationDidBecomeActive' is the last delegate callback and needs to build up the UI.
 		if !didSetupUI && !appLaunchedFromUserActivityURL {
 			setupUI()
-			showUI()
-
-			appLaunchedFromUserActivityURL = false
-			didSetupUI = true
-			route = nil
+			showUI {
+				self.appLaunchedFromUserActivityURL = false
```

Comment:
Just thinking out loud; Would it make sense to move this into `showUI`?

Reply:
I didn't even realize the content of the completion closure was exactly the same 😄 Makes sense, moved it 👍🏼